README.md

```diff
@@ -49,3 +49,33 @@ gulp serve
 Serve task will create a proxy server on port 3000 that goes via Django’s server on port 8000 enhancing it with
 additional features provided by [Browsersync](http://www.browsersync.io/), such as CSS/JS reload, interaction
 synchronisation and much more.
+
+## Releasing
+
+### Releasing to non-production
+
+> Currently, `staging` is the only non-production environment.
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/fala) for the feature branch.
+1. Copy the `feature_branch.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
+    ```
+    Pushing tag for rev [3e1a4ef9ecf6] on {https://registry.service.dsd.io/v1/repositories/fala/tags/github-config.843dcc7}
+    ```
+1. [Deploy `feature_branch.<sha>`](https://ci.service.dsd.io/job/DEPLOY-fala/build?delay=0sec).
+    * `ENVIRONMENT` is the target environment, select "staging".
+    * `DEPLOY_BRANCH` is the branch that needs to be released (`github-config` in the above example).
+    * `VERSION` is either `latest` for the last successful build on that branch, or a specific 7-character prefix of the Git SHA (`843dcc7` in the above example).
```

Comment:
For this repository, these variables seem to be different:

- `version` seems to be the concatenated `branch.sha` format
- `deploy_branch` seems to be correlating to the deploy repo's branch name.

<img width="232" alt="screen shot 2018-08-30 at 15 38 14" src="https://user-images.githubusercontent.com/1526295/44858820-c36cd300-ac6a-11e8-8756-99777d4bf15a.png">